### PR TITLE
Move Infoblox from Provisioning to Installing

### PR DESCRIPTION
**Pull request by Lennonka**
Until we have a guide for networking, this will be a better place for Infoblox DHCP and DNS setup.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into: N/A

### CODE REVIEW

**Comment by Lennonka** [comment]
I'm assuming that Inflobox cannot be used in disconnected environments.

**Comment by ekohl** [comment]
> I'm assuming that Inflobox cannot be used in disconnected environments.

I don' think that's true. AFAIK Infoblox is an on-premise appliance and I don't see why it could be used.

**Comment by Lennonka** [comment]
In that case yes. I didn't know they had on-prem solution as well.

**Comment by Lennonka** [comment]
I've made a couple of improvements.

File: guides/common/modules/proc_installing-the-infoblox-ca-certificate.adoc [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
 # update-ca-trust extract
 ----
++
+The `_infoblox.example.com_` entry must match the host name for the Infoblox application in the X509 certificate.
 
-* The `_infoblox.example.com_` entry must match the host name for the Infoblox application in the X509 certificate.
-
-To test the CA certificate, use a CURL query:
+.Verification
+* Test the CA certificate by using a CURL query:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-the-infoblox-ca-certificate.adoc:20]:
```suggestion
* Test the CA certificate by using a `curl` query:
```

I am unsure about CURL. See https://curl.se/

**Review by maximiliankolb — COMMENTED** [review]
I cannot comment much on Infoblox itself, but the diff LGTM

File: guides/common/modules/con_infoblox-limitations.adoc [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
-{SmartProxyServer} communicates with a single Infoblox node using the standard HTTPS web API.
+{ProductName} communicates with a single Infoblox node using the standard HTTPS web API.
 If you want to configure clustering and High Availability, make the configurations in Infoblox.
 
-Hosting PXE-related files using Infoblox's TFTP functionality is not supported.
-You must use {SmartProxy} as a TFTP server for PXE provisioning.
-For more information, see xref:Configuring_Networking_{context}[].
+Hosting PXE-related files by using the TFTP functionality of Infoblox is not supported.
+You must use {ProductName} as a TFTP server for PXE provisioning.
+For more information, see {ProvisioningDocURL}Configuring_Networking_{context}[Configuring networking] in _{ProvisioningDocTitle}_.
```
Comment (Lennonka) [inline guides/common/modules/con_infoblox-limitations.adoc:12]:
```suggestion
For more information, see {ProvisioningDocURL}Configuring_Networking_provisioning[Configuring networking] in _{ProvisioningDocTitle}_.
```

**Pull request merged**